app/controllers/concerns/caching_headers.rb

```diff
@@ -21,6 +21,8 @@ def set_cache_control_headers(
 
     request.session_options[:skip] = true # no cookies
 
+    RequestStore.store[:edge_caching_in_place] = true # To be observed downstream.
```

Comment:
🔥

Reply:
I think we should surface this gem in the `Gemfile`, right now it's a transitive dependency of `rails-settings-cached` but as any of the other gems that we use directly, it should be listed in the `Gemfile`. Something like:

```ruby
gem 'request_store', '~> 1.5' # RequestStore gives you per-request global storage
```

Reply:
@rhymes 👍